lib/moesif_rack/moesif_middleware.rb

```diff
@@ -236,13 +219,28 @@ def call env
         # Perform the API call through the SDK function
         begin
           @random_percentage = Random.rand(0.00..100.00)
+
+          # Event UserId
+          user_id = !event_model.user_id.nil?  ? event_model.user_id : nil
```

Comment:
Not super familiar with ruby. Is `user_id = event_model.user_id` functionally equivalent?

Reply:
Refactored with PR #15